engine/admin/logging/logentries.md

```diff
@@ -0,0 +1,51 @@
+---
+aliases: ["/engine/reference/logging/logentries/"]
+title: "Logentries logging driver"
+description: "Describes how to use the logentries logging driver."
+keywords: ["logentries, docker, logging, driver"]
+---
+
+# Logentries logging driver
+
+The `logentries` logging driver sends container logs to the
+[Logentries](https://logentries.com/) server.
+
+## Usage
+
+Some options are supported by specifying `--log-opt` as many times as needed:
+
+ - `logentries-token`: specify the logentries log set token
+
+Configure the default logging driver by passing the
+`--log-driver` option to the Docker daemon:
+
+```console
```

Comment:
I found this: https://github.com/jneen/rouge/wiki/List-of-supported-languages-and-lexers

Apparently `bash` is an alias for `shell`, and `console` is an alias for `shell_session`. Probably it's not the end of the world, but I guess sometimes you have to choose  a standard and stick to it!